### ISSUE TITLE
Option to show playback queue at the bottom of the track info panel

### ISSUE THREAD

**Issue by jansol** [Taiko2k/TauonMusicBox#1073]
I practically never have reason to open the playlist panel, however I normally have the track info panel open. I do also use the playback queue a lot and it would be handy to have it visible - the most natural place being at the bottom of the track info, maybe with the addition of making that panel scrollable.

**Comment by Solarunit** [comment]
>I normally have the track info panel open. I do also use the playback queue a lot and it would be handy to have it visible - the most natural place being at the bottom of the track info, maybe with the addition of making that panel scrollable.

I use queue a lot too (but I also use playlists panel)

>the most natural place being at the bottom of the track info

do you mean right panel where the cover art is? Can you share example screenshot?

**Comment by jansol** [comment]
Yeah, on the right side, like this:
![image](https://github.com/Taiko2k/TauonMusicBox/assets/2588851/330f0bfc-13ff-4f65-bf06-2705673e3383)

**Comment by Solarunit** [comment]
I see. Maybe it make sense to put it there.
I use this right panel to showcase only album art in full height and nothing else.

**Comment by Movisivi** [comment]
> Yeah, on the right side, like this: ![image](https://private-user-images.githubusercontent.com/2588851/292026613-330f0bfc-13ff-4f65-bf06-2705673e3383.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ1MTUzNDQsIm5iZiI6MTcwNDUxNTA0NCwicGF0aCI6Ii8yNTg4ODUxLzI5MjAyNjYxMy0zMzBmMGJmYy0xM2ZmLTRmNjUtYmYwNi0yNzA1NjczZTMzODMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMDZUMDQyNDA0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjQyNmU4MzQ5NWVjMWYxMGM3MjY2NjUzYjE0NzY1MWMwNmU2ZGJjNjczOTk2NGNmY2MyOTg5MzhhZjE0YmEyNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lLjRjlfQu69zcNABeGp68pYeYl_neM-shVFgcRUg-cg)

Yes this is amazing @Taiko2k please consider it doing it like this. Way better than what I suggested https://github.com/Taiko2k/TauonMusicBox/issues/880

**Comment by Taiko2k** [comment]
Thing is that area isn't really designed for it, like in that screenshot there's not a lot of room, like there would only be room for 1 item in the queue, unless the window was bigger.
